### DISCUSSION TITLE
Add support for Approve Sign In to Apple Watch app

### DISCUSSION THREAD

**Discussion by dewinterjack** [community/community#12002] [Mobile]
When a sign in request is sent to the iOS app, my apple watch shows the notification but does not let me approve it without opening my phone. 

Could a button be added to the notification to approve the sign in?

The approve sign in from the mobile app has been a great feature. It would be nice to improve it.

**Comment by stevepopovich** [comment]
Hi Jack! This would be cool to have. Passing along to the team for prioritization!

**Reply by koalazub** [reply]
Hey! Just checking in to see if this has progressed at all?

**Reply by stevepopovich** [reply]
We don't have any current plans to add this feature, but it is in our backlog.

**Comment by obriencj** [comment]
Recently started using Unifi's Verify for 2FA in their ecosystem, and was surprised that a similar feature was available there immediately, allowing me to approve sign-in requests straight from my watch. It really would help ease the move into 2FA if this feature existed for github as well!

**Reply by Macaulay1234** [reply]
Yes

**Comment by Macaulay1234** [comment]
> Hi Jack! This would be cool to have. Passing along to the team for prioritization!

**Comment by Macaulay1234** [comment]
> Recently started using Unifi's Verify for 2FA in their ecosystem, and was surprised that a similar feature was available there immediately, allowing me to approve sign-in requests straight from my watch. It really would help ease the move into 2FA if this feature existed for github as well!

**Comment by Macaulay1234** [comment]
> When a sign in request is sent to the iOS app, my apple watch shows the notification but does not let me approve it without opening my phone. 
> 
> Could a button be added to the notification to approve the sign in?
> 
> The approve sign in from the mobile app has been a great feature. It would be nice to improve it.
